### DISCUSSION TITLE
3.0.0

### DISCUSSION THREAD

**Discussion by the-database** [the-database/mpv-upscale-2x_animejanai#30] [Announcements]
- Add new HD V3 models
- Update to [mpv-lazy-2024V0](https://github.com/hooke007/MPV_lazy/releases/tag/20240102)
- Update to [mpvnet v7.1.0.0](https://github.com/mpvnet-player/mpv.net/releases/tag/v7.1.0.0) 
- Add [AnimeJaNaiConfEditor](https://github.com/the-database/AnimeJaNaiConfEditor), launch with `ctrl+E`
- Add script to view upscaling status, toggle with `ctrl+J`

<hr /><em>This discussion was created from the release <a href='https://github.com/the-database/mpv-upscale-2x_animejanai/releases/tag/3.0.0'>3.0.0</a>.</em>

**Comment by FelixKainz** [comment]
Awesome! How does the performance compare? Is it tuned to improve the performance/quality ratio or just push it further in one direction, hurting the other? From the new readme, it sounds to push further towards quality.

**Reply by 422415** [reply]
It still uses Compact, UltraCompact, SuperUltraCompact models. Performance should be the same as V2.

**Comment by trikats** [comment]
When extracting "mpv-upscale-2x_animejanai-full-package-3.0.0" in Windows 11 I get an unexpected error is keeping you from copying the folder.
Had to click skip multiple times for the "input" and "output" .pb files.
![Screenshot 2024-01-20 001402](https://github.com/the-database/mpv-upscale-2x_animejanai/assets/157194953/0600494b-6aa1-4695-9164-81a508364986)

**Comment by Lulu11041** [comment]
Is it just me or this version break thumbfast and Command Palette?
